rust/connlib/shared/src/control.rs

```diff
@@ -21,32 +21,16 @@ use tokio_tungstenite::{
     tungstenite::{self, handshake::client::Request},
 };
 use tungstenite::Message;
-use url::Url;
 
 use crate::{get_user_agent, Error, Result};
+use phoenix_channel::LoginUrl;
 
 const CHANNEL_SIZE: usize = 1_000;
 const HEARTBEAT: Duration = Duration::from_secs(30);
```

Comment:
```suggestion
// Heartbeat sent from client, gateway, or relay to the API server (Not the same as the portal?)
const HEARTBEAT: Duration = Duration::from_secs(30);
```

I ran into this on another project - There were like 3 different heartbeats, so it was useful to just always declare the sender and receiver. In this case I don't know if the heartbeat behavior differs between:

- Endpoints (clients, gateways) talking to the control plane / API
- Clients / gateways talking to relays
- Clients talking to gateways
- Anything else

Reply:
This is just clients to portal but will soon be deleted.